src/lib/efreet/meson.build

```diff
@@ -56,6 +56,7 @@ install_headers(efreet_header_src,
   install_dir : dir_package_include,
 )
 
+if not sys_windows
```

Comment:
Could you explain to me the point of disabling this?

Reply:
Do you think we should manage to build `deprecated_efreed*`?

For now it throws the following error:
```
FAILED: src/lib/efreet/efreet_trash-1.dll src/lib/efreet/efreet_trash-1.pdb
"lld-link"  /MACHINE:x64 /OUT:src/lib/efreet/efreet_trash-1.dll  "/nologo" "/DEBUG" "/PDB:src/lib/efreet\efreet_trash-1.pdb" "/DLL" "/IMPLIB:src/lib/efreet\efreet_trash.lib" "src/lib/efreet\efreet.lib" "src/lib/eina\ein
a.lib" "src/lib/evil\evil.lib" "subprojects\getopt\libgetopt.a" "subprojects\pcre-8.37\libpcre.a" "subprojects\pcre-8.37\libpcreposix.a" "src/lib/eo\eo.lib" "src/lib/efl\efl.lib" "psapi.lib" "ole32.lib" "ws2_32.lib" "se
cur32.lib" "uuid.lib" "kernel32.lib" "user32.lib" "gdi32.lib" "winspool.lib" "shell32.lib" "ole32.lib" "oleaut32.lib" "uuid.lib" "comdlg32.lib" "advapi32.lib"
lld-link: error: <root>: undefined symbol: _DllMainCRTStartup

FAILED: src/lib/efreet/efreet_mime-1.dll src/lib/efreet/efreet_mime-1.pdb
"lld-link"  /MACHINE:x64 /OUT:src/lib/efreet/efreet_mime-1.dll  "/nologo" "/DEBUG" "/PDB:src/lib/efreet\efreet_mime-1.pdb" "/DLL" "/IMPLIB:src/lib/efreet\efreet_mime.lib" "src/lib/efreet\efreet.lib" "src/lib/eina\eina.l
ib" "src/lib/evil\evil.lib" "subprojects\getopt\libgetopt.a" "subprojects\pcre-8.37\libpcre.a" "subprojects\pcre-8.37\libpcreposix.a" "src/lib/eo\eo.lib" "src/lib/efl\efl.lib" "psapi.lib" "ole32.lib" "ws2_32.lib" "secur
32.lib" "uuid.lib" "kernel32.lib" "user32.lib" "gdi32.lib" "winspool.lib" "shell32.lib" "ole32.lib" "oleaut32.lib" "uuid.lib" "comdlg32.lib" "advapi32.lib"
lld-link: error: <root>: undefined symbol: _DllMainCRTStartup
```

Reply:
I actually have no idea about it, it just sounds strange removing it so it doesn't break XD

But as it is deprecated...It should be ok, right?

Reply:
Yeah, that's all about it. heh